src/main/java/seedu/manager/ui/Ui.java

```diff
@@ -14,26 +16,28 @@ public class Ui {
     private static final String SEPARATOR = "------------------------";
 
     private final Scanner userInput;
+    private final PrintStream utf8;
 
     /**
      * Constructs a new Ui
      */
     public Ui() {
         this.userInput = new Scanner(System.in);
+        this.utf8 = new PrintStream(System.out, true, StandardCharsets.UTF_8);
     }
 
     /**
      * Greets user upon program startup
      */
     public void greetUser() {
-        System.out.println(WELCOME_MESSAGE);
+        utf8.println(WELCOME_MESSAGE);
     }
```

Comment:
May I understand why `utf8` rather than `System.out` is being used here?

Reply:
because system.out does not uses utf-8 and if im not wrong default is ascii. utf8 is jus an attribute to print it in utf8 encoding we can name it to something like writer if yall want

Reply:
I think it's okay to just use utf8 as the name of the stream, thanks for the clarification!